front/src/modules/ui/data/data-table/components/DataTableColumnDropdownMenu.tsx

```diff
@@ -14,19 +13,21 @@ export type DataTableColumnDropdownMenuProps = {
   isFirstColumn: boolean;
   isLastColumn: boolean;
   primaryColumnKey: string;
+  dropdownScopeId: string;
 };
 
 export const DataTableColumnDropdownMenu = ({
   column,
   isFirstColumn,
   isLastColumn,
   primaryColumnKey,
+  dropdownScopeId,
 }: DataTableColumnDropdownMenuProps) => {
   const { handleColumnVisibilityChange, handleMoveTableColumn } =
     useTableColumns();
 
   const { closeDropdown } = useDropdown({
-    dropdownScopeId: ColumnHeadDropdownId,
```

Comment:
as this code is called from within DropdownScope it should not be needed. useDropdown will find it by itself